src/components/bars/SSIHeaderBar/index.tsx

```diff
@@ -62,6 +65,24 @@ const SSIHeaderBar: FC<Props> = (props: Props): JSX.Element => {
     dispatch<any>(logout());
   };
 
+  const onDeleteWallet = async (): Promise<void> => {
+    setShowProfileMenu(false);
+    const activeUser: IUser = store.getState().user.activeUser!;
+
+    props.navigation.navigate(MainRoutesEnum.POPUP_MODAL, {
+      title: translate('profile_delete_wallet_action_title'),
+      details: format(translate('profile_delete_wallet_action_subtitle'), `${activeUser.firstName} ${activeUser.lastName}`),
```

Comment:
It would be nicer if we adjust the translate method to accept a vararg for variables

Reply:
refactored it as i18n could already do this